src/partials/_animal--elephant.hbs

```diff
@@ -0,0 +1,29 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 639.43 726.34" width="350" class="c-elephant">
+  <defs>
+    <style>
+      .elephant-1{fill:#4c5a52;}.elephant-2{fill:#3a3f36;}.elephant-3,.elephant-6{fill:#5e6e66;}.elephant-4{fill:#444d3e;}.elephant-5{fill:#fff;}.elephant-6{stroke:#8b9b92;stroke-width:2px;}.cls-7{fill:#8b9b92;}
```

Comment:
Since this is an inline SVG, this CSS could be a part of your Sass files if you’d like.